.github/workflows/qa.yaml

```diff
@@ -30,7 +30,7 @@ env:
 jobs:
   go-sanity:
     name: "Go: Code sanity"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
I wonder if we shouldn't do similar to what we did for adsys: https://github.com/ubuntu/adsys/pull/1065/commits/293224d67481a0dee4f5cd2f874f294cedb83cec

Reply:
I would use `DEFAULT_CI_RUNNER` though, as we may use different ones in a matrix of ubuntu versions.

Reply:
I was about to do it but I also don't have the required permissions :/

Reply:
As stated in the channel we're using `runs-on: ubuntu-24.04 # ubuntu-latest-runner` since github doesn't support using `${{ env.FOO }}` expansion here, and using per-project variables is too restrictive.

So, going with that.